ytsync/ytsync.go

```diff
@@ -182,6 +195,9 @@ func (s *Sync) FullCycle() (e error) {
 	}
 
 	s.db = redisdb.New()
+	s.videosMapMux.Lock()
```

Comment:
why is this code all the way down here, so far from line 125 where syncedVideos is first declared?

Reply:
the only reason i put it there is because there were a bunch of assignments there and i wanted to keep them grouped, but I agree it's not very read-able this way. I'll move them up